CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 <!--## Unreleased-->
+- `build`: Update automatic `webcomponentsjs` polyfilling to move it and all affected elements following it into the body so that the `custom-elements-es5-adapter.js` can work properly in IE11. (See [#627](https://github.com/Polymer/polymer-cli/issues/627))
```

Comment:
We really need some docs on transformations like this. Can you describe this in the "build" section of the README for now, and maybe we'll find a new place later if that isn't good.